packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -56,6 +57,44 @@ keywords: [menu, dropdown, action menu]
 
 The MenuTrigger accepts exactly two children: the element which triggers the opening of the Menu and the Menu itself. The trigger element must be the first child passed into the MenuTrigger and should support press events.
 
+## Long press
+By default, a MenuTrigger's Menu is opened by pressing the trigger element or activating it via the <Keyboard>Space</Keyboard> or <Keyboard>Enter</Keyboard> keys. However, there may be cases in which your trigger element
+should perform a separate default action, such as selection, on press and should only display the Menu when long pressed. This behavior can be set by providing "longPress" to the `trigger` prop. Now, the Menu will only be opened upon
+pressing and holding the trigger element or by using the <Keyboard>Option</Keyboard> (<Keyboard>Alt</Keyboard> on Windows) + <Keyboard>Down Arrow</Keyboard> keys while focusing the trigger element.
+
+The example below illustrates how one would setup a MenuTrigger to have long press behavior.
+
+```tsx example
+import CloneStamp from '@spectrum-icons/workflow/CloneStamp';
+import Crop from '@spectrum-icons/workflow/Crop';
+import CropRotate from '@spectrum-icons/workflow/CropRotate';
+import Slice from '@spectrum-icons/workflow/Slice';
+import {Text} from '@react-spectrum/text';
+
+<MenuTrigger trigger="longPress">
+  <ActionButton
+    aria-label="Crop tool"
+    onPress={() => alert('Cropping!')}>
+    <Crop />
+  </ActionButton>
+  <Menu>
+    <Item textValue="Crop Rotate">
+      <CropRotate />
+      <Text>Crop Rotate</Text>
+    </Item>
+    <Item textValue="Slice">
+      <Slice />
+      <Text>Slice</Text>
+    </Item>
+    <Item textValue="Clone stamp">
+      <CloneStamp />
+      <Text>Clone Stamp</Text>
+    </Item>
+  </Menu>
+</MenuTrigger>
+```
+```
```

Comment:
This makes the whole events section into a code block

Reply:
🤦, mea culpa